packages/client/tests/functional/_utils/setupTestSuiteClient.ts

```diff
@@ -110,7 +110,7 @@ export async function setupTestSuiteClient({
   const clientGenOptions: GenerateClientLegacyOptions & GenerateClientESMOptions = {
     datamodel: schema,
     schemaPath,
-    binaryPaths: { libqueryEngine: {} },
+    binaryPaths: {},
```

Comment:
Why do we still have `binaryPath` here btw? Because a generator can declare a dependency on the schema engine?